app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryTask.js

```diff
@@ -35,7 +35,9 @@ class ExpandedStoryTask extends Component {
   }
 
   render() {
-    const { story, onToggle, onDelete } = this.props;
+    const { story, onToggle, onDelete, disabled } = this.props;
+
+    if (!story.tasks.length && disabled) return null;
```

Comment:
Can we check disabled first and then the length of the tasks as we do for the other components? Just to keep it standard 😄